src/main/java/com/linecorp/armeria/server/docs/EnumInfo.java

```diff
@@ -47,21 +48,24 @@ static EnumInfo of(EnumMetaData enumMetaData) {
             }
         }
 
-        return new EnumInfo(enumClass.getName(), constants);
+        final String name = enumClass.getName();
+        return new EnumInfo(name, constants, docStrings.get(name));
     }
 
-    static EnumInfo of(String name, List<Object> constants) {
-        return new EnumInfo(name, constants);
+    static EnumInfo of(String name, List<Object> constants, Map<String, String> docStrings) {
+        return new EnumInfo(name, constants, docStrings.get(name));
```

Comment:
Yeah I guess it must be, if so it's ok. Surprised since the enum name and package name are displayed separately in the docservice UI, didn't realize that name is split() inside ClassInfo (a bit bizarre).